.github/CODEOWNERS

```
@@ -2,7 +2,7 @@
 ##### Global Protection Rule ######
 ###################################
 # NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below.
-*                                               @jbair06 @ramto @SvetBorislavov 
+*                                               @hashgraph/transaction-tool @jbair06 @ramto @SvetBorislavov 
```

Remove `jbair06` as they are part of the `transaction-tool` team.

```suggestion
*                                               @hashgraph/transaction-tool @ramto @SvetBorislavov 
```